kitsune/search/v2/management/commands/es7_reindex.py

```diff
@@ -78,12 +79,20 @@ def handle(self, *args, **kwargs):
 
             before = kwargs["updated_before"]
             after = kwargs["updated_after"]
+            if not before and not after:
+                after = config.DEFAULT_REINDEX_UPDATED_AFTER.get(dt.__name__)
             if before or after:
```

Comment:
not part of the pr but I would love if we could remove all these `if` statements for `before` and `after` or structure them differently by chaining the query.
```python
if before:
    qs = qs.filter(before)
if after:
   qs = qs.filter(after)
```
We could also move the `try/except` into the manager.

probably this comment falls under the nit category